autotest/pst_from_tests.py

```diff
@@ -607,7 +600,7 @@ def add_py_function_test(tmp_path):
 def test_mf6_freyberg(tmp_path):
     # import sys
     # sys.path.insert(0, os.path.join("..", "..", "pypestutils"))
-    import pypestutils as ppu
+    ppu = pytest.importorskip("pypestutils")
```

Comment:
we probably don't want to be skipping this test if import fails. 

Currently, there is the pyemu geostats interp fallback if ppu is not available (though this may be deprecated in the future if we want to lean more on ppu)

So here, I think we either want to raise if import fails or continue (which would mean that we are testing the pyemu fallback).